src/main/java/dk/dbc/lobby/rest/ApplicantsResource.java

```diff
@@ -224,4 +262,38 @@ public Response getApplicantBody(@PathParam("id") String id) {
         final StreamingOutput streamingOutput = outputStream -> outputStream.write(applicantBodyEntity.getBody());
         return Response.ok().type(applicantEntity.getMimetype()).entity(streamingOutput).build();
     }
+
+    private Response getStreamingResponse(List<ApplicantEntity> resultSet) {
```

Comment:
No changes, only refactored out into function since it was to be used now by two functions.